modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaServiceTrackerListCheck.testjava

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.change.tracking.internal;
+
+import com.liferay.change.tracking.internal.helper.CTTableMapperHelper;
```

Comment:
@michaelmdelcavalcanti, do we really need all this content in the test's scope?

Can we simplify to use only what is really necessary for we verify that the implemented check is working as intended?